src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -31,14 +31,16 @@ enum CORINFO_InstructionSet
     InstructionSet_Rcpc=13,
     InstructionSet_VectorT128=14,
     InstructionSet_Rcpc2=15,
-    InstructionSet_ArmBase_Arm64=16,
-    InstructionSet_AdvSimd_Arm64=17,
-    InstructionSet_Aes_Arm64=18,
-    InstructionSet_Crc32_Arm64=19,
-    InstructionSet_Dp_Arm64=20,
-    InstructionSet_Rdm_Arm64=21,
-    InstructionSet_Sha1_Arm64=22,
-    InstructionSet_Sha256_Arm64=23,
+    InstructionSet_Sve=16,
```

Comment:
Any reason why we cannot add new entries at the bottom and not touch other entries?

Reply:
This file is autogenerated and the new entry is "at the bottom"

The generator basically generates:
* `Isa1`
* `Isa2`
* `...`
* `Isa1_But64Bit`
* `Isa2_But64Bit`
* `..._But64Bit`